README.md

```diff
@@ -68,10 +68,10 @@ pip install astro-sdk-python[amazon,google,snowflake,postgres]
     @aql.transform()
     def top_five_animations(input_table: Table):
         return """
-            SELECT Title, Rating
+            SELECT title, rating
```

Comment:
Should we add a description on README about https://deeplearning.lipingyang.org/2017/01/07/postgresql-column-names-of-a-table-are-case-sensitive/ in case of PostGres like you mentioned on slack? It might be useful for user

Reply:
@tatiana There should also be communicated on how are we converting columns from files to database respective col names.

Reply:
@sunank200 @utkarsharma2 This is a sqlite example, which is supposed to be very tiny - a quick start - without explaining further detail, just something users can copy, paste and see working. Where in our docs do you think would be a more suitable place to add this?